frontends/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:20.12-alpine
+LABEL maintainer "ODL DevOps <mitx-devops@mit.edu>"
+
+COPY . /src
+
+WORKDIR /src
+
+RUN yarn set version 4.1.1
+RUN yarn install
```

Comment:
Should this be `yarn install --immutable`?

Reply:
Updated.